packages/components/src/__future__/Tag/Tag/_docs/MigrationGuide.mdx

```diff
@@ -0,0 +1,168 @@
+- `dismissable` prop has been removed.
+  - A `DismissableTag` component is now available in its place.
+- `Avatar` and all child content must now be composed by the consumer.
+  - This offers a greater degree of flexibility to ensure that the right accessible context can be provided to the end user.
+
+
+## Migration examples
+
+The following are example of migrating from previously available props to the composable solution.
+
+### Status
+
+With the `variant` prop there are four value associated with statuses in the legacy tag:  `statusLive`, `statusDraft`, `statusClosed`, `statusAction`.
+
+For the most part these can all be replaced easily with the use of the `color` prop, with exception to the `live` status.
+
+#### Before
+
+<Canvas of={LegacyTagStories.Status} />
+
+
+#### After
+
+<Canvas of={TagMigrationStories.StatusMigration} sourceState="shown"/>
+
+While the `LiveIcon` is exported from `@kaizen/components` the legacy `Tag` used an animated implementation, which is not currently available. As an interim solution you can use the following story's code snippet to create a `LiveIconComponent`. This can then be passed into as `Children` of the `Tag`.
+
+<Canvas of={TagMigrationStories.LiveIconComponentStory} sourceState="hidden"/>
+
+### Validation
+
+Following the same pattern as status, `validationPositive`, `validationInformative`, `validationNegative`, `validationCautionary` can all be replace with the `color` and `icon` prop.
+
+Note: because the icons convey meaning, they should have an accessible label that provides context to the tag. A contextual `aria-label`and role of `img` addresses one of the [accessibility issues](https://github.com/cultureamp/kaizen-discourse/issues/102) with the legacy Tag.
+
+#### Before
+
+<Canvas of={LegacyTagStories.Validation} />
+
+#### After
+
+<Canvas of={TagMigrationStories.ValidationMigration} sourceState="shown"/>
+
+
+Note: When using the `icon` prop, the icon's color will be determined by `color` value that has been set. This is to ensure that and accessible contrast ratio is achieved.
+
+Any Icons added as a child of the `Tag` will need to be manually set to ensure the right contrast ratio.
+
+### Sentiment
+
+While migration of any `Tag` using, `sentimentPositive`, `sentimentNeutral` or `sentimentNegative` is relatively straight forward using the `color` prop, `sentimentNone` does not align to any Kaizen's [semantic colors](https://cultureamp.atlassian.net/wiki/spaces/DesignSystem/pages/3064857902/Color).
+
+#### Before
+
+<Canvas of={LegacyTagStories.Sentiments} />
+
+#### After
+
+<Canvas of={TagMigrationStories.SentimentsMigration} sourceState="shown"/>
+
+If required, you could use `classNameOverride` to implement the same UI. It is important that any override of the default styles still meets minimum contrast ratios to ensure the component is accessible.
+
+<Canvas of={TagMigrationStories.SentimentNone} />
```

Comment:
Perhaps the correct suggestion is to actually use one of the provided colours. Check with the designers though :)